src/service/RelayService.ts

```diff
@@ -4,22 +4,33 @@ import {sleep} from "../util/utils";
 import {NostrEvent} from "../model/NostrEvent";
 
 export class RelayService {
-    relay = new Relay('wss://relay.nostr.band')
+    public relayList: Relay[];
```

Comment:
This currently will break the functionality. Find a way to pass this relay list in. An env variable would be optimal.